contracts/src/v0.8/upkeeps/CronUpkeep.sol

```diff
@@ -40,16 +40,19 @@ contract CronUpkeep is KeeperCompatibleInterface, KeeperBase, ConfirmedOwner, Pa
 
   event CronJobExecuted(uint256 indexed id, uint256 timestamp);
   event CronJobCreated(uint256 indexed id, address target, bytes handler);
+  event CronJobUpdated(uint256 indexed id, address target, bytes handler);
   event CronJobDeleted(uint256 indexed id);
 
   error CallFailed(uint256 id, string reason);
   error CronJobIDNotFound(uint256 id);
+  error ExceedsMaxJobs();
   error InvalidHandler();
   error TickInFuture();
   error TickTooOld();
   error TickDoesntMatchSpec();
 
   address immutable s_delegate;
+  uint256 public immutable s_maxJobs;
```

Comment:
This should also be added in delegate?

Reply:
I think that because the val is `immutable`, it's not stored in contract storage. And thus doesn't need to be in the delegate.

Reply:
Actually on the delegate contract s_delegate is not marked as immutable. How is it working o.O